src/sentry/api/endpoints/project_plugins.py

```diff
@@ -13,6 +13,6 @@ def get(self, request, project):
         context = serialize([
             plugin
             for plugin in plugins.configurable_for_project(project, version=None)
-            if plugin.has_project_conf()
+            if plugin.has_project_conf() or plugin.get_plugin_type() == 'release-tracking'
```

Comment:
I don't see a problem with just adding another method to the plugin to feature detect this.

We can even do something like,

```python
def has_plugin_conf(self):
  return self.has_project_conf()
```

As the default behavior. Then your subclass can override.